integration/common.sh

```diff
@@ -29,6 +29,20 @@ waitForServerStartup() {
   sleep 2
 }
 
+pickUnusedPort() {
+  local base=$1
+  local port
+  while :; do
+    for (( port = "${base}" ; port <= 61000 ; port++ )); do
+      result=$(netstat -a | grep ":${port}" || true)
```

Comment:
Suggestion:

```bash
if ! lsof -i :$port > /dev/null; then
  echo $port
  break
fi
```

The idea is that with lsof we don't need to grep (it returns 1 if the port is unused).